styles/npm/catppuccin.user.css

```diff
@@ -1098,19 +1098,28 @@
       }
     }
 
+    /* pricing page */
```

Comment:
You gotta move this section to the lines/place I mentioned too.

Reply:
ima do just that